HOWTO_RELEASE.md

```diff
@@ -57,7 +54,6 @@ paste, but please pay attention to details!
     git tag -s X.Y.Z
     git push --tags
 
-
```

Comment:
It's good to have two lines before titles, helps the eye separation. It's innocuous when rendered, too.